beancount_n26/__init__.py

```diff
@@ -235,40 +245,62 @@ def extract(self, file_, existing_entries=None):
             for index, line in enumerate(reader):
                 meta = data.new_metadata(file_.name, index)
 
-                s_amount_eur = self._translate('amount_eur')
-                s_amount_foreign_currency = self._translate(
-                    'amount_foreign_currency'
-                )
-                s_payee = self._translate('payee')
-                s_payment_reference = self._translate('payment_reference')
-                s_type_foreign_currency = self._translate(
-                    'type_foreign_currency'
-                )
+                postings = []
 
-                if line[s_amount_eur]:
-                    amount = Decimal(line[s_amount_eur])
-                    currency = 'EUR'
-                else:
-                    amount = Decimal(line[s_amount_foreign_currency])
+                if line[s_amount_foreign_currency]:
+                    exchange_rate = Decimal(line[s_exchange_rate])
+                    amount_eur = Decimal(line[s_amount_eur])
+                    amount_foreign = Decimal(line[s_amount_foreign_currency])
                     currency = line[s_type_foreign_currency]
 
```

Comment:
There's no such assert for the `self.account` field. And since the field is mandatory in the importer configuration, I don't think it would be relevant

Reply:
Ah, I missed that. In that case we should make the parameter optional. Not everyone is making multi-currency transactions so it should ideally be something that we provide for those who want it.

For instance, I've never had to make a multi-currency transaction so I wouldn't know what to pass as a value for this parameter. If we make it mandatory I'd most likely put in `None` here, in which case we might as well make it optional.

Reply:
Fixed in 2b165b8f3740802fd5709cea6eb84d48a9937549, field is now optional & is only checked when we have non-0 fees on a transaction